app/views/internal/shared/_navbar.html.erb

```diff
@@ -0,0 +1,11 @@
+<% menu_items = %w[comments articles users tags welcome broadcasts reports pages tools chat_channels growth mods].each_with_object({}) { |v, h| h[v] = v } %>
```

Comment:
As I said, I feel bad for this, but I'm not sure what to do with the menu in terms of design to fit all the items.
First, I thought about creating a webhooks list in `administrate`, but there were some issues with the namespaced models and I didn't want to spend a lot of time figuring them out.

Reply:
I think it's okay to not have it in `administrate`. I believe we're going to be sunsetting it eventually and solely relying on `/internal` in the future.

Reply:
Yeah, I was keeping it in mind, so didn't want to invest much time into it.